### PR TITLE
Integration: Chromatic Decorator [LG-166] 

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

**Review by bruugey — APPROVED** [review]
bravooooo @TheSonOfThomp

File: packages/button/src/Button.story.tsx [hunk 1]
```diff
@@ -68,19 +98,73 @@ export const Playground: StoryFn<ButtonProps> = ({
...
+export const XSmallSize: StoryType<typeof Button> = () => <></>;
+XSmallSize.parameters = {
+  generate: {
+    args: {
+      size: Size.XSmall,
+    },
+  },
+};
+
+export const Loading: StoryType<typeof Button> = () => <></>;
```
Comment (shaneeza) [inline packages/button/src/Button.story.tsx:150]: Nothing shows up in this story in SB, is that correct?

**Review by shaneeza — APPROVED** [review]

File: packages/button/src/Button.story.tsx (hunk 1 above)
Comment (spark33) [reply packages/button/src/Button.story.tsx:150]: Yeah we disabled it to avoid issues with the spinner

**Pull request merged**